colossalai/tensor/colo_parameter.py

```diff
@@ -34,5 +37,12 @@ def from_torch_tensor(tensor: torch.Tensor,
         tensor.__init__(tensor, requires_grad=requires_grad, spec=spec)
         return tensor
 
+    def add_linked_modules(self, module: torch.nn.Module):
```

Comment:
what are linked modules?
The concept is weird.

Reply:
If is a link, name as module_link

Reply:
It's used to detect all modules a ColoParam bound to. E.g. XXXembedding and XXXclassifier share the same weight, then the param weight will have two linked_modules {XXXembedding, XXXclassifier}

Reply:
Ohh, you mean sharded_param_modules? The word link indicates 2 modules link to the same params. Too indirectly.

Reply:
I see.

Reply:
Done.